lib/widgets/home_page/simulation_test.dart

```diff
@@ -12,7 +13,7 @@ class SimulationTestWidget extends StatelessWidget {
   List<Map<String, dynamic>> topics = [
     {
       "title": "Mini Test",
-      "subtitle": "70 questions",
+      "subtitle": "70 Questions",
```

Comment:
not translated, u can try : 

WITH PARAMETER
in json file -> "questions" : "{} Questions"
in this line -> 'questions'.tr(args : ["70"])

WITHOUT PARAMETER (HARDCODE)
in json file -> "questions" : "70 Questions"
in this line -> 'questions'.tr()